Resources/Locale/en-US/chat/managers/chat-manager.ftl

```diff
@@ -13,8 +13,15 @@ chat-manager-whisper-headset-on-message = You can't whisper on the radio!
 chat-manager-server-wrap-message = SERVER: {"{0}"}
 chat-manager-sender-announcement-wrap-message = {$sender} Announcement:
                                                 {"{0}"}
-chat-manager-entity-say-wrap-message = {$entityName} says: "{"{0}"}"
-chat-manager-entity-whisper-wrap-message = {$entityName} whispers: "{"{0}"}"
+chat-manager-entity-death-message = {$entityName}'s body seizes up and falls limp, his eyes dead and lifeless...
+chat-manager-entity-scream-message = {$entityName} screams!
+chat-manager-entity-say-wrap-message = {$entityName} says, "{"{0}"}"
+chat-manager-entity-ask-wrap-message = {$entityName} asks, "{"{0}"}"
+chat-manager-entity-yell-wrap-message = {$entityName} yells, "{"{0}"}"
+chat-manager-entity-shout-wrap-message = {$entityName} shouts, "{"{0}"}"
+chat-manager-entity-scream-wrap-message = {$entityName} screams, "{"{0}"}"
+chat-manager-entity-exclaims-wrap-message = {$entityName} exclaims, "{"{0}"}"
+chat-manager-entity-whisper-wrap-message = {$entityName} whispers, "{"{0}"}"
```

Comment:
This is unnecessary - you can pass it in the same way you do `entityName`

Reply:
im pretty sure this isn't unnecessary. this is just how wrap messages work